### PR TITLE
Add `Sinatra::HamlHelpers` to sinatra-contrib

### PR DESCRIPTION
**Pull request by dentarg**
Brings back three helpers from Haml 5:
https://github.com/haml/haml/blob/v5.2.2/lib/haml/helpers.rb#L294-L357

### CODE REVIEW

**Comment by dentarg** [comment]
Haml 6 retained some helpers methods, but only for Rails: https://github.com/haml/haml/blob/v6.2.3/lib/haml/rails_helpers.rb

It is possible to use those, if you monkey-patch `String` a little bit: https://github.com/Starkast/wikimum/pull/395/files#diff-8d6558cd6fe082484754565c92a04af566daa4bfc3ee0b8cdeb541e04dceefec

File: sinatra-contrib/lib/sinatra/haml_helpers.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require 'sinatra/base'
+require 'sinatra/capture'
+
+module Sinatra
+  # = Sinatra::HamlHelpers
+  #
+  # This extension provides some of the helper methods that existed in Haml 5
+  # but was removed in Haml 6. To use this in your app, just +register+ it:
```
Comment (olleolleolle) [inline sinatra-contrib/lib/sinatra/haml_helpers.rb:10]:
```suggestion
  # but were removed in Haml 6. To use this in your app, just +register+ it:
```

**Review by olleolleolle — APPROVED** [review]
Looking good!

File: sinatra-contrib/lib/sinatra/haml_helpers.rb (hunk 1 above)
Comment (dentarg) [reply sinatra-contrib/lib/sinatra/haml_helpers.rb:10]: Thanks

**Comment by olleolleolle** [comment]
Thanks for keeping this useful!

**Pull request merged**